.github/workflows/Linux.yml

```diff
@@ -89,6 +90,29 @@ jobs:
         with:
           compiler: ${{ matrix.compiler }}
 
+      - name: Install lcov
+        if: matrix.compiler != 'default'
+        run: |
+          sudo apt-get install \
+            libcapture-tiny-perl \
+            libdatetime-perl \
+            libdatetime-format-dateparse-perl \
+            libgd-perl \
+            libgd3 \
+            libjbig0 \
+            libjpeg-turbo8 \
+            libjpeg8 \
+            libjson-perl \
+            libperlio-gzip-perl \
+            libtiff5 \
+            libwebp7
+
+          git clone https://github.com/linux-test-project/lcov.git -b v2.0
```

Comment:
What's this?

Reply:
lcov can be installed with apt-get afaik

Reply:
I need version 2.0 for compatibility with macOS. I do not want to maintain two configurations.